ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_fft.py

```diff
@@ -163,3 +163,54 @@ def test_jax_numpy_ifft(
         atol=1e-02,
         rtol=1e-02,
     )
+
+
+# ifft2
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.ifft2",
+    dtype_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("complex"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```

This should always be `valid` - if only `complex` data-type is supported by the function then that's what `valid` will return.

Reply:
> This should always be `valid` - if only `complex` data-type is supported by the function then that's what `valid` will return.

Dear @hello-fri-end, just changed it to valid. Any other comments?